lib/mlcatconfig/test_Config.py

```diff
@@ -0,0 +1,21 @@
+import pytest
+import configparser
+import os
+from lib.mlcatconfig.driver_path import *
+
+def test_config_class():
+    
+    path=os.path.abspath("mlcat.cfg")
+    t_config=Config('t_mailbox')
+    t_config.read(path)
```

Comment:
please add the case where mlconfig comes from the user and is in a different directory. Also include the case where there are white space characters in the path.